doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -7,9 +7,7 @@ An Open Virtual Appliance (OVA) is a Virtual Machine (VM) image that consists of
 ***********************
 Recommended Hypervisors
 ***********************
-All of the following hypervisors should work: `VMware ESX Server <https://www.vmware.com/products/esxi-and-esx.html>`_,
-`VMware vSphere <https://www.vmware.com/products/vsphere.html>`_,
```

Comment:
Are we planning to remove support for VMware ESx and VMware Fusion. I guess it's working on both.

Reply:
Ok. This may contradictory as in `Tested by:` section we mentioned few people validated old OVA on VMare Vfusion.

`Oct 5, 2020: Andriy Tkachuk (andriy.tkachuk@seagate.com) using OVA release 1.0.2 by running VMWare Fusion 11.`

Hope this is fine.

Reply:
done